content/posts/newsletter/Newsletter for 2020-01-10.md

```diff
@@ -0,0 +1,167 @@
+
+![The Ecosystem is Moving](https://framapic.org/vnIr36vc8UVR/V2BVA5KG8ozl.png)
+
+Alex Gnauck wrote about  [Simpler code in handlers using pattern matching](https://www.ag-software.net/2019/12/30/simpler-code-in-handlers-using-pattern-matching/) in MatriX.
+
+During the 36C3 in December 2019, a bunch of people finalized the idea to form a [SIG dedicated to end-to-end encryption](https://mail.jabber.org/pipermail/standards/2019-December/036769.html).
+
+### Newsletter translations 
+
+* [French](https://linuxfr.org/news/lettre-d-information-xmpp-3-decembre-2019-xmpp-dans-toutes-les-langues)
+* [German](https://www.jabber.de/xmpp-newsletter-xmpp-auf-allen-sprachen-03-dez-2019/)
+* [Spanish](https://www.jabber.de/es-xmpp-newsletter-xmpp-en-todos-los-idiomas-03-dic-2019/)
+
+## Software releases 
+
+### Servers 
+
+The Ignite Realtime community has released:
+
+* [Openfire 4.5.0 Beta](https://discourse.igniterealtime.org/t/openfire-4-5-0-beta-release/86888)
+* [Fastpath Openfire plugin 4.5.0 Beta](https://discourse.igniterealtime.org/t/fastpath-openfire-plugin-4-5-0-beta-release/86895)
+
+### Clients and applications
+
+Anu Pokharel the developer of Monal has made significant changes to its [MUC implementation](https://monal.im/blog/group-chat-improvements/) which improves the situation for iOS ([4.1 and 4.2](https://monal.im/blog/ios-4-1-and-4-2/) and [4.2 final beta](https://monal.im/blog/ios-4-2-final-beta/)) and macOS ([2.5 beta 1](https://monal.im/blog/mac-2-5-beta-1/) [2.5 beta 2](https://monal.im/blog/mac-2-5-beta-2/)) users. Furthermore there has been many [bug fixes](https://monal.im/blog/cve-fixes/), including [HTTP upload](https://monal.im/blog/http-upload/), and [the end of iOS9 support](https://monal.im/blog/the-end-of-ios-9-support/).
+
+![Monal iOS](https://framapic.org/ZyopsYd66GsN/aOp2n1IrXfIy.png)
+
+Gajim has posted their [Development News](https://gajim.org/post/2020-01-07-development-news-december/) for December 2019: old message dialog system, compact tooltip when hovering a group chat in Gajim’s contact list, new assistant for password change and account removal, HTTP File Upload now work asynchronously with the ‘libsoup’ library, bug fixes, plugin updates, and changes in python-nbxmpp.
+
+Tigase has released [BeagleIM 3.4 and Siskin IM 5.4](https://tigase.net/beagleim-3.4-and-siskin-5.4-released/): BeagleIM 3.4 allows you to set MUC room avatar, simplifies MUC room settings, and bug fixes, while SiskinIM 5.4 adds support for setting MUC room avatar.
+
+[Movim 0.16.1](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/cdfc0a4c-3459-4d3b-8c15-08994810d54e) has been released: you can now share posts to your connected chatrooms, communities layout were redesigned, all the messages that you sent in the one to one discussions can now be edited, the videoconferencing feature was heavily refactored, and the bug fixes.
```

Comment:
`layout` is singular but you use `were`. I'm not sure whether you meant `communities layouts were redesigned`, `communities layout was redesigned` or even `community layouts were redesigned`

Reply:
Instead of `and the bug fixes` I would write `and bugs fixed`.

Reply:
Sorry, it's a copy-paste: I think it's "communities layout was redesigned".